runtime/compiler/optimizer/FearPointAnalysis.cpp

```diff
@@ -35,16 +35,26 @@
 #include "infra/Checklist.hpp"
 
 
-bool TR_FearPointAnalysis::virtualGuardsKillFear(TR::Compilation *comp, TR::Node *virtualGuardNode)
+bool TR_FearPointAnalysis::virtualGuardKillFear(TR::Compilation *comp, TR::Node *virtualGuardNode)
    {
+   if (comp->cg()->supportsMergingGuards())
```

Comment:
I think this condition is backwards and we need `!supportsMergingGuards()`

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/16354/commits/d2a57e7eaec47151b5e7794df65185ae1cd5fead